docs/guides/chat-notifications.md

```diff
@@ -0,0 +1,183 @@
+Please see the [get started guide](../get-started/index.md)
+or the [installation guide](installation.md).
+
+The GitOps toolkit controllers emit Kubernetes events whenever a resource status changes.
+You can use the [notification-controller](../components/notification/controller.md)
+to forward these events to Slack, Microsoft Teams, Discord or Rocket chart.
+The notification controller is part of the default toolkit installation.
+
+
+## Define a provider
+
+First create a secret with your incoming webhook:
+
+```sh
+kubectl -n flux-system create secret generic webhook-url \
+--from-literal=address=https://webhooks.com/services/YOUR/WEBHOOK
+```
+
+Note that the secret must contain an `address` field,
+
+it can be a Slack, Microsoft Teams, Discord, Google Chat or Rocket webhook URL.
+
+Create a notification provider for your service by referencing the above secret:
+
+=== "Slack"
+    ```bash
+    $ flux create alert-provider slack \
+      --type slack \
+      --secret-ref webhook-url \
+      --channel general \
+      --export > slack-provider.yaml
+    ---
+    # slack-provider.yaml
+    apiVersion: notification.toolkit.fluxcd.io/v1beta1
+    kind: Provider
+    metadata:
+      name: slack
+      namespace: flux-system
+    spec:
+      type: slack
+      channel: general
+      secretRef:
+        name: webhook-url
+    ```
+
+=== "Discord"
+
+    ```bash
+    $ flux create alert-provider discord \
+      --type discord \
+      --secret-ref webhook-url \
+      --channel general \
+      --username flux \
+      --export > discord-provider.yaml
+    ---
+    # discord-provider.yaml
+    apiVersion: notification.toolkit.fluxcd.io/v1beta1
+    kind: Provider
+    metadata:
+      name: discord
+      namespace: flux-system
+    spec:
+      channel: notifications-alerts-webhooks-example
+      secretRef:
+        name: webhook-url
+      type: discord
+      username: notifications-alerts-webhooks-example-bot
+
+    ```
+=== "Microsoft Teams"
+    ```bash
+    $ flux create alert-provider teams \
+      --type msteams \
+      --secret-ref webhook-url \
+      --channel general \
+      --export > teams-provider.yaml
+    ---
+    # teams-provider.yaml
+    apiVersion: notification.toolkit.fluxcd.io/v1beta1
+    kind: Provider
+    metadata:
+      name: teams
+      namespace: flux-system
+    spec:
+      channel: general
+      secretRef:
+        name: webhook-url
+      type: msteams
+    ---
+=== "Google Chat"
+    ```bash
+    $ flux create alert-provider google-chat \
+      --type googlechat \   
+      --secret-ref webnook-url \
+      --export > gchat-provider.yaml
+    ---
+    # gchat-provider.yaml
+    apiVersion: notification.toolkit.fluxcd.io/v1beta1
+    kind: Provider
+    metadata:
+      name: google-chat
+      namespace: flux-system
+    spec:
+      secretRef:
+        name: webhook-url
+      type: googlechat
+    ```
+
+The provider type can be `slack`, `discord`, `msteams`, `rocket`,  `googlechat` or `generic`
```

Comment:
As of flux2 0.12 we also support Sentry and Webex.

Reply:
```suggestion
The provider type can be `slack`, `discord`, `msteams`, `googlechat`,`rocket`,`webex` or `generic`
```